### DISCUSSION TITLE
update version  1.47.x

### DISCUSSION THREAD

**Discussion by antoniocarlosbotelho** [alisonrafael/metabase-db2-driver#9] [General]
Metabase 47 has got something in it for everyone—CSV uploads, PDF downloads, searching, serialization, and more.

**Comment by alisonrafael** [comment]
tivemos um _pull request_ em #10 onde recebemos uma atualização no driver que funciona na versão 51 do metabase

**Reply by antoniocarlosbotelho** [reply]
Show, funcionando perfeitamente .
